docs/install_guide_macos.rst

```diff
@@ -0,0 +1,217 @@
+	* Qt version zvoliť Qt 5.x
+	* CMake Tool zvoliť na CMake zo záložky CMake
+	* Vpravo hore klikúť na Make Default
+	* Potvrdiť tlačidlom OK
+
+Build & Run konfigurácia:
+-------------------------
+
+* V ľavom menu Qt Creatora kliknúť na Projects
+* Kliknúť na Build pri unity
+* Edit build configuration zvoliť na Debug
+* Build directory zvoliť na ~/3dsoftviz/build
+* Pri položke Build steps kliknúť na Details a zaškrtnúť install_unity
+* Kliknutím na Build project (kladivo) projekt zbuildovať
+* Naľavo kliknúť na Run pri unity
+* Pri Run configuration rozbaliť Add menu a zvoliť Custom Executable
+* Zvoliť binárku 3dsoftviz/_install/bin/3DSoftviz
+* Working directory nastaviť na 3dsoftviz/_install/bin
+
+*
+Známe problémy, riešenia a rady
+-------------------------------
+
+Q: Build error  "/usr/local/.//mkspecs/macx-clang"
+**************************************************
+
+::
+
+	CMake Error at /usr/local/lib/cmake/Qt5Core/Qt5CoreConfig.cmake:15 (message):
+	  The imported target "Qt5::Core" references the file
+
+	     "/usr/local/.//mkspecs/macx-clang"
+
+**A:** Chyba pravdepodobne je na strane Homebrew, pri inštalácii Qt5 nevytvorí dve symlinky (ani po zavolaní brew link). Musíte ich vytvoriť ručne (ak treba, upravte verziu v ceste, návod bol robený pri nainštalovanej 5.9.2):
+
+::
+
+	ln -s /usr/local/Cellar/qt/5.9.2/mkspecs /usr/local/mkspecs
+	ln -s /usr/local/Cellar/qt/5.9.2/plugins /usr/local/plugins
+
+Ak príkaz zlyhá, pretože symlinky už existujú, zmažte ich a skúste znova.
+
+Q: Build error v 3dsoftviz/src/Mouse3d/LibMouse3d/Mac, riadok 68: 	Mouse3dMacDeviceGlob::ptr->OnMove( motionData );
+********************************************************************************************************************
+
+::
+
+	/Users/peto.marusin/3d/src/Mouse3d/LibMouse3d/Mac/Mouse3dMacDevice.cpp:68: error: no viable conversion from 'vector<float>' to 'const vector<double>'
+	            Mouse3dMacDeviceGlob::ptr->OnMove( motionData );
+	                                               ^~~~~~~~~~
+	/Library/Developer/CommandLineTools/usr/include/c++/v1/vector:530:5: note: candidate constructor not viable: no known conversion from 'std::vector<float>' to 'initializer_list<value_type>' (aka 'initializer_list<double>') for 1st argument
+	    vector(initializer_list<value_type> __il);
+	    ^
+	/Library/Developer/CommandLineTools/usr/include/c++/v1/vector:542:5: note: candidate constructor not viable: no known conversion from 'std::vector<float>' to 'const std::__1::vector<double, std::__1::allocator<double> > &' for 1st argument
+	    vector(const vector& __x);
+	    ^
+	/Library/Developer/CommandLineTools/usr/include/c++/v1/vector:548:5: note: candidate constructor not viable: no known conversion from 'std::vector<float>' to 'std::__1::vector<double, std::__1::allocator<double> > &&' for 1st argument
+	    vector(vector&& __x)
+    ^
+
+**A:** Zakomentovať riadok (TEMPORARY!)
+
+Q: Runtime lua error ast.lua:9: module 'metrics.lookup' not found
```

Comment:
toto sa vyriesi novym luametric modulom, ked to bude na gitlab-e

Reply:
ok, odstranim z doc